packages/hop-node/src/config/validation.ts

```diff
@@ -68,32 +69,76 @@ export async function validateConfig (config?: FileConfig) {
     Chain.Polygon
   ]
 
+  const validTokenKeys = [
+    'USDC',
+    'USDT',
+    'DAI',
+    'ETH',
+    'MATIC',
+    'WBTC'
+  ]
+
+  let enabledChains: string[] = []
   if (config.chains) {
-    const networkKeys = Object.keys(config.chains)
-    await validateKeys(validNetworkKeys, networkKeys)
+    enabledChains = Object.keys(config.chains)
+    validateKeys(validNetworkKeys, enabledChains)
+    const validChainKeys = ['rpcUrl', 'waitConfirmations']
+    for (const chain in config.chains) {
+      validateKeys(validChainKeys, Object.keys(config.chains[chain]))
+      if (!config.chains[chain]) {
+        throw new Error(`RPC config for chain "${chain}" is required`)
+      }
+      const { rpcUrl, waitConfirmations } = config.chains[chain]
+      if (!rpcUrl) {
+        throw new Error(`RPC url for chain "${chain}" is required`)
+      }
+      try {
+        const parsed = new URL(rpcUrl)
+        if (!parsed.protocol || !parsed.host || !['http:', 'https:'].includes(parsed.protocol)) {
+          throw new URIError()
+        }
+      } catch (err) {
+        throw new Error(`rpc url "${rpcUrl}" is invalid`)
+      }
+      if (waitConfirmations == null) {
+        throw new Error(`waitConfirmations for chain "${chain}" is required`)
+      }
+      if (typeof waitConfirmations !== 'number') {
```

Comment:
See comment above about checking for the value `0` here.

Reply:
It checks now if `waitConfirmations >= 0`. Resolved with https://github.com/hop-protocol/hop/pull/346/commits/36a93478a3aa1d9fccd0c560ea29b4a32c1059bb